activator/steplib/activate.go

```diff
@@ -138,8 +92,14 @@ func copyStepYML(libraryURL, id, version, dest string) error {
 func ListCachedStepVersions(log stepman.Logger, stepLib models.StepCollectionModel, stepLibURI, stepID string) []string {
 	versions := []models.Semver{}
 
-	for version, step := range stepLib.Steps[stepID].Versions {
-		_, err := activateStep(stepLib, stepLibURI, stepID, version, step, log, true)
```

Comment:
This was doing a full step activation just to check the error. I replaced this with a proper stepman call

Reply:
Oh, true. The reason I had to replace this is the circular dependency between the `activateStep()` function that now calls this function.